yb-voyager/src/utils/utils.go

```diff
@@ -425,3 +425,12 @@ func GetEnvAsInt(key string, fallback int) int {
 	}
 	return int(valueInt)
 }
+
+func GetMapKeysSorted(m map[string]*string) []string {
+	keys := make([]string, 0, len(m))
+	for key := range m {
+		keys = append(keys, key)
+	}
```

Comment:
Nit: let's bring the `lo` package and use the `Keys()` method from it (https://github.com/samber/lo#keys)

The `lo` package will be beneficial in many other places.